setup.py

```diff
@@ -55,7 +55,7 @@ def get_version(file="flake8_isort.py"):
     include_package_data=True,
     zip_safe=False,
     install_requires=[
-        'flake8 >= 3.2.1, <5',
+        'flake8 >= 3.2.1, <6',
```

Comment:
maybe we can drop the upper limit? I would say yes

Reply:
Hmm, as my view, I think we should keep it.
We don't know about what happend when flake8 bump to version 6.
So I think we need verify again when major change is better.

But if you think remove is okay. I will help remove it.

Reply:
It's a good checkpoint, as long as updates come fast enough to account for it.

Reply:
I would recommend removing the upper bound, you're just going to have this same scramble every time I release a major version and I'm going to have to deal with the grumpy users every time that report stuff to the wrong place

Reply:
The problem seems to be that 3.0.0 of flake8-isort was released without upper bound, so when a new major release of flake8 comes along, that one is dug up by dependency resolution and all bug fixes between 3.0.0 and now are resurfaced. I agree an upper bound is a good checkpoint if it prevents dependencies from being satisfied until the new version is confirmed to be compatible. But when a 2+ year old release surface it is pretty confusing.

Reply:
generally I find the best approach for libraries is to only include `<` when there is a known broken version and even then generally prefer `!=` with the assumption it will be fixed